messenger.rst

```diff
@@ -623,6 +623,32 @@ config and start your workers:
 
 See the `Supervisor docs`_ for more details.
 
+
+.. tip::
+
+    To get a deterministic worker name like for e.g. the Redis transport, you may pass the process number as an
+    environment variable.
+
+    .. code-block:: ini
+
+        ;/etc/supervisor/conf.d/messenger-worker.conf
+        [program:messenger-consume]
+        […]
+        environment = REDIS_CONSUMER=consumer-%(process_num)d
+
+    You can then use it in your messenger configuration:
+
+    .. configuration-block::
+
+        .. code-block:: yaml
+
+            # config/packages/messenger.yaml
+            framework:
+                messenger:
+                    transports:
+                        redis: 'redis://localhost:6379/messages/symfony/%env(REDIS_CONSUMER)%'
```

Comment:
`REDIS_CONSUMER` Environment variable not found? How I can get this variable from supervior configuration?

Reply:
It is in the code blog just before. 

https://github.com/symfony/symfony-docs/pull/11869/files#diff-cefce3d23e1437ef5885ad7d486ec8d57efd7a3f6ad55bd5997bb202b537ff59R637

Reply:
@Nyholm What I did is: I added the line `environment = REDIS_CONSUMER=consumer-%(process_num)d` to the configuration file `/etc/supervisor/conf.d/messenger-worker.conf`, now my problem is in Symfony application the variable `REDIS_CONSUMER` is not defined, so how I can defined it?